services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -513,6 +588,7 @@ async def search_files_starting_with(
                 project_ids=can_read_projects_ids,
                 file_id_prefix=prefix,
                 partial_file_id=None,
+                only_files=True,
```

Comment:
so I think this should also paginate. Please create an issue and assign to @pcrespov and yourself...